iocage/Config/Jail/File/Fstab.py

```diff
@@ -247,15 +248,28 @@ def parse_lines(
                 )
                 continue
 
+            source = fragments[0].strip()
             destination = fragments[1].strip()
 
             # skip lines with destinations overlapping self.basejail_lines
             if destination in basejail_destinations:
                 continue
 
+            _backup_prefix = "backup:///"
```

Comment:
this seems like a new feature / different bug

should this be here?

Reply:
In exports the jails root dataset is replaced with `backup:///` in the fstab file. It could either be detected here or a new type `AbsolutePath` would be required. Replacing it here seemed to be less difficult, although now the logic is spread in two different places.